### DISCUSSION TITLE
Sandboxie-Plus 1.10.2 false positives

### DISCUSSION THREAD

**Discussion by deeperfacts** [sandboxie-plus/Sandboxie#3147] [General]
![1 10 2](https://github.com/sandboxie-plus/Sandboxie/assets/141161684/cf027812-1351-4623-8f21-874480ffe2f1)
![1 10 2-2](https://github.com/sandboxie-plus/Sandboxie/assets/141161684/85b0e809-784c-45d2-82d0-c8d281d62252)

**Comment by bastik-1001** [comment]
Those are most likely false positives, due to the circumstance of the installer for the pre-release not being signed.

To a degree it is valid to find that suspicious, since some unsigned installer tries to install a driver (that is signed) and there is a history of signed installers, if any vendor ever keeps track of this. For an installed security solution, it may also look suspicious that an installer replaces previously signed files with unsigned files, if there is an upgrade installation.

**Reply by bastik-1001** [reply]
The rule for "sandboxdetect" should be triggered by the installer as it detects if Sandboxie is active.

The rule about "taskkill" is triggered as this tool gets executed to terminate processes that are running (like sandman.exe) to install a newer version.

BTW, I don't know what naming scheme is applied by CrowdStrike Falcon, but including "confidence_100%" in the string, better be accurate.

**Reply by deeperfacts** [reply]
@bastik-1001 Thank you for explanations.
